lib/jazzy/sourcekitten.rb

```diff
@@ -120,41 +121,65 @@ def self.process_undocumented_token(doc, declaration)
       make_default_doc_info(declaration)
     end
 
-    def self.parameters_from_xml(xml)
-      xml.xpath('Parameters/Parameter').map do |parameter_el|
-        {
-          name: XMLHelper.xpath(parameter_el, 'Name'),
-          discussion: Jazzy.markdown.render(
-              XMLHelper.xpath(parameter_el, 'Discussion') || '',
-            ),
-        }
+    def self.make_paragraphs(doc, key)
+      return nil unless doc[key]
+      doc[key].map do |p|
+        if p.keys.include?('Para')
+          Jazzy.markdown.render(p['Para'])
+        elsif p.keys.include?('Verbatim')
+          Jazzy.markdown.render("```\n" + p['Verbatim'] + "```\n")
```

Comment:
yes, but we don't have that info at all from `Verbatim` tags anyway, plus if we're going to do our own ReST definition parsing, this is only temporary. This code path is only triggered for the contents of a ReST definition (practically `:param:` & `:returns:`). The comment body still goes straight through redcarpet.